docs/modules/serialization/pages/schema-evolution.adoc

```diff
@@ -0,0 +1,703 @@
+== Introducing schema evolution
+:description: This topic introduces schema evolution in Hazelcast Compact serialization and explains how to safely evolve object schemas across rolling upgrades, clients, and persisted clusters without data loss or downtime. It covers backward and forward compatibility, deterministic writes, versioned maps, MapStore and persistence considerations, SQL and index evolution, and best practices for zero-downtime migrations in distributed environments.
+
+{description}
+
+Schema evolution refers to the ability of Hazelcast to handle changes in the structure of serialized objects without requiring downtime, full data reloads, or strict synchronization across cluster members and clients.
```

Comment:
what does this mean? `strict synchronization across cluster members and clients`